app/uk/gov/hmrc/merchandiseinbaggage/views/layouts/govukLayout.scala.html

```diff
@@ -51,9 +54,14 @@
 @headerDefault = {
 @headerBlock.getOrElse {
     @govukHeader(Header(
-        homepageUrl = Some("https://www.gov.uk/"),
+        homepageUrl = Some(messages("service.name.none.a.href")),
+        serviceUrl = declarationType match {
```

Comment:
i think you can remove this match and simply use `Some(messages("service.name.$declarationType.a.href"))` instead ?

Reply:
I have removed the `match` as requested